mitodl/mit-open
# Offerer banner UI

### What are the relevant tickets?

Closes [#4454](https://github.com/mitodl/hq/issues/4454)



### Description (What does it do?)
This PR adds a banner background, logo, copy text, and nav/breadcrumb element to offeror pages. It also includes a default template configuration for offeror pages that gets loaded in when running ```./manage.py backpopulate_resource_channels```

### Screenshots:
<img width="1519" alt="Screenshot 2024-06-05 at 8 58 38 AM" src="https://github.com/mitodl/mit-open/assets/196425/3b07f261-502c-4373-a385-4f978d8fccc7">
<img width="520" alt="Screenshot 2024-06-05 at 8 58 56 AM" src="https://github.com/mitodl/mit-open/assets/196425/6098ed8b-858d-4887-be6b-7cdaa1bed837">


### How can this be tested?
To test this:
- checkout this branch
- make sure you have offerors loaded ```./manage.py update_offered_by```
- run ```./manage.py backpopulate_resource_channels --overwrite```
- visit offerer channel pages and see that they match the mocks:
  - http://od.odl.local:8063/c/offeror/xpro
  - http://od.odl.local:8063/c/offeror/see
  - http://od.odl.local:8063/c/offeror/mitx
  - http://od.odl.local:8063/c/offeror/ocw
  - http://od.odl.local:8063/c/offeror/bootcamps
  - http://od.odl.local:8063/c/offeror/mitpe


### Additional Context
